### PR TITLE
Bugfix/creatable validation

### PR DESCRIPTION
**Pull request by fortunato**

### CODE REVIEW

File: src/components/Form/Form.tsx [hunk 1]
```diff
@@ -5,14 +5,39 @@ import customRenderers from '../customRenderers';
...
 	onChange: OnChangeCallback;
 };
 
 export const DEFAULT_RENDERERS = [...materialRenderers, ...customRenderers, ...customLayoutRenderers];
 
+// The oneOf property in the JsonSchema we use to populate the select options does is used for validation purposes
+// If an option for a creatable field does not exist in the oneOf array, it is deemed an invalid option
+// What we do here is adding a new option in case a non-existent option is presenting itself
+// The altered schema is then used for validation purposes but NOT rendering purposes
+function addCreatableSupportToSchema(schema: CustomJsonSchema, data: Record<string, any>): CustomJsonSchema {
```
Comment (johan-hage) [inline src/components/Form/Form.tsx:20]: Do we have some test for this?

File: src/index.hbs [hunk 2]
```diff
@@ -9,6 +9,12 @@
 	<script src="https://static.amsterdam.nl/fonts/mtiFontTrackingCode.min.js"></script>
 	<link href="https://static.amsterdam.nl/fonts/fonts.css" rel="stylesheet" />
 	<title>DMS Upload</title>
+	<style>
+		/* Ensure that the react-select dropdown is on top in case it is present inside a modal */
+		body > div:last-child {
```
Comment (johan-hage) [inline src/index.hbs:14]: not better to address it specificly in the style of the creatable by id? as now we have no way of knowing that the last div "always" contains the creatable. In the future we can add components, which also have a div on the end of the page.

**Review by johan-hage — CHANGES_REQUESTED** [review]
Works as expected :+1: Some comments.

**Review by johan-hage — APPROVED** [review]
Nice work with the test coverage.

File: src/components/Form/Form.tsx (hunk 1 above)
Comment (fortunato) [reply src/components/Form/Form.tsx]: Valid point :see_no_evil:. Fixed

File: src/index.hbs (hunk 2 above)
Comment (fortunato) [reply src/index.hbs]: Solved it from the bmi-component-library side via a prop

**Pull request merged**